more/free-programming-interactive-tutorials-en.md

```diff
@@ -111,6 +111,7 @@
 
 ### Data Science
 
+* [Data Science Foundations - IBM SkillBuild](https://skillsbuild.org/students/course-catalog/data-science) *(Email login reqired to access the course)*
```

Comment:
An insignificant misspelling in notes: `reqired`

but better normalize format according to CONTRIBUTING guidelines.

`*(email address required)*`

:rocket::rocket: